src/Support/Http/Controllers/GraphQLController.php

```diff
@@ -26,7 +29,12 @@ public function __invoke(
             new StartRequest($request)
         );
 
-        $operationOrOperations = $requestParser->parseRequest($request);
+        try {
+            $operationOrOperations = $requestParser->parseRequest($request);
+        } catch (JsonException|RequestError $e) {
+            throw new BadRequestHttpException('Error parsing GraphQL request', $e);
```

Comment:
This erases valuable information

Reply:
Why? I pass the original exception to `BadRequestHttpException` so it is logged too.

Reply:
The error message shown to the user will be just that of the parent.

Reply:
Uhm, currently it is showing just "error 500" for the production site and full exception details (with nested exceptions) for the dev. 

Anyways, should I add this PR to the `laragraph/utils`?

Reply:
Yeah, I would appreciate a PR there.